terraform/bambictf.tf

```diff
@@ -196,6 +204,34 @@ systemctl start wg-quick@internal
 TERRAFORMEOF
 }
 
+resource "hcloud_server" "moloch" {
+  name        = "moloch${count.index + 1}"
+  image       = data.hcloud_image.bambimoloch.id
+  location    = "fsn1"
+  server_type = local.moloch_type
+  count       = local.moloch_count
+
+  ssh_keys = data.hcloud_ssh_keys.all_keys.*.id
+
+  provisioner "local-exec" {
+    command = "curl --user \"${local.ovh_dyndns_username}:${var.ovh_dyndns_password}\" \"https://www.ovh.com/nic/update?system=dyndns&hostname=${self.name}.${local.ovh_dyndns_domain}&myip=${self.ipv4_address}\""

```

Comment:
Note to self, we manually need to create that DynDNS entry in the OVH webinterface before we can change it